images/virtualization-artifact/pkg/controller/vd/internal/source/http.go

```diff
@@ -195,8 +195,7 @@ func (ds HTTPDataSource) Sync(ctx context.Context, vd *virtv2.VirtualDisk) (bool
 
 		source := ds.getSource(vd, supgen)
 
-		wffc := vd.Spec.BindingMode != nil && *vd.Spec.BindingMode == virtv2.VirtualDiskBindingModeWaitForFirstConsumer
-		err = ds.diskService.Start(ctx, diskSize, vd.Spec.PersistentVolumeClaim.StorageClass, source, vd, supgen, wffc)
+		err = ds.diskService.Start(ctx, diskSize, vd.Spec.PersistentVolumeClaim.StorageClass, source, vd, supgen)
```

Comment:
ErrDefaultStorageClassNotFound is not internal error, write message to condition and return nil

Reply:
As well as `storageProfile not found`